tools/mo/requirements.txt

```diff
@@ -1,6 +1,5 @@
 -c ../constraints.txt
 numpy>=1.16.6
+importlib-metadata; python_version < "3.8" and sys_platform == "linux"
 networkx
-defusedxml
```

Comment:
why is it removed?

Reply:
It must have been an accident. Thank you for finding it, it shouldn't have been removed. I reverted it.